flutter_vlc_player/example/lib/single_tab.dart

```diff
@@ -1,70 +1,83 @@
 import 'dart:io';
-import 'dart:typed_data';
 
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_vlc_player/flutter_vlc_player.dart';
+import 'package:flutter_vlc_player_example/video_data.dart';
+import 'package:flutter_vlc_player_example/vlc_player_with_controls.dart';
 import 'package:path_provider/path_provider.dart';
 
-import 'video_data.dart';
-import 'vlc_player_with_controls.dart';
-
 class SingleTab extends StatefulWidget {
   @override
   _SingleTabState createState() => _SingleTabState();
 }
 
 class _SingleTabState extends State<SingleTab> {
-  VlcPlayerController _controller;
+  static const _networkCaching = 2000;
+  static const _fontSize = 30;
+  static const _height = 400.0;
+
   final _key = GlobalKey<VlcPlayerWithControlsState>();
 
+  VlcPlayerController _controller;
   //
   List<VideoData> listVideos;
   int selectedVideoIndex;
 
   Future<File> _loadVideoToFs() async {
     final videoData = await rootBundle.load('assets/sample.mp4');
     final videoBytes = Uint8List.view(videoData.buffer);
-    var dir = (await getTemporaryDirectory()).path;
-    var temp = File('$dir/temp.file');
+    final dir = (await getTemporaryDirectory()).path;
+    final temp = File('$dir/temp.file');
     temp.writeAsBytesSync(videoBytes);
+
     return temp;
   }
 
   void fillVideos() {
     listVideos = <VideoData>[];
     //
-    listVideos.add(VideoData(
-      name: 'Network Video 1',
-      path:
-          'http://samples.mplayerhq.hu/MPEG-4/embedded_subs/1Video_2Audio_2SUBs_timed_text_streams_.mp4',
-      type: VideoType.network,
-    ));
+    listVideos.add(
+      VideoData(
```

Comment:
Maybe let's turn this into a list literal instead of appending elements one by one?